docker/cami-medical-compliance/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM williamyeh/ansible:ubuntu16.04
```

Comment:
Yes, it's a base image with Ansible installed. Ansible do not maintain such images themselves anymore so we're using what's most popular.